hugo.yaml

```diff
@@ -5,8 +5,6 @@ title: Jaeger
 disableKinds: [taxonomy]
 theme: [basetheme, jaeger-docs]
 disableAliases: true # We do redirects via Netlify's _redirects file
-pygmentsCodeFences: true
-pygmentsUseClasses: true
```

Comment:
These settings come from 7 years ago, see:

- https://github.com/jaegertracing/documentation/pull/35

These settings haven't been used by Hugo for quite a while now, so it's more than time to drop them :).